### DISCUSSION TITLE
balance/balance4-ha

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#170] [General]
# balance/balance4-ha

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/balance/balance4-ha.html

**Comment by anaer** [comment]
1. 文中使用的边缘路由，而图片中为边界路由 是否可以统一下
2. `多个四层负载均起以相同的 BGP 权重`  负载均衡器

**Reply by isno** [reply]
整体写完的时候，会再统一名词、错误别字、段落之类的。 不过这期间发现错误，可以给我提。

**Comment by zy09** [comment]
有个疑问，我在实际使用中看到很多都用的fullnat ，然后做一个snat 池提升并发上线，tunnel 的方式不论硬件负载还是软件负载在实际使用中并不多，考虑到排错和维护的难度。
